scripts/deployment/bridges/wormhole/test/globals_celo.json

```diff
@@ -1 +1 @@
-{"contractVerification":true,"useLedger":false,"derivationPath":"m/44'/60'/2'/0/0","providerName":"celoAlfajores","gasPriceInGwei":"10","networkURL":"https://alfajores-forno.celo-testnet.org","sourceGovernorChainId":"10002","L1WormholeRelayerAddress":"0x7B1bD7a6b4E61c2a123AC6BC2cbfC614437D0470","timelockAddress":"0x00000000000000000000000014CF2e543AB75B321bcf84C3AcC88d570Ccf9106","L2WormholeRelayerAddress":"0x306B68267Deb7c5DfCDa3619E22E9Ca39C374f84","wormholeMessengerAddress":"0x118173028162C1b7c6Bf8488bd5dA2abd7c30F9D","childMockERC20Address":"0x34235f9D447f9F54167e2Ac7A0F4283cB3fAD669"}
\ No newline at end of file
+{"contractVerification":true,"useLedger":false,"derivationPath":"m/44'/60'/2'/0/0","providerName":"celoAlfajores","gasPriceInGwei":"10","networkURL":"https://alfajores-forno.celo-testnet.org","sourceGovernorChainId":"10002","L1WormholeRelayerAddress":"0x7B1bD7a6b4E61c2a123AC6BC2cbfC614437D0470","timelockAddress":"0x00000000000000000000000014CF2e543AB75B321bcf84C3AcC88d570Ccf9106","L2WormholeRelayerAddress":"0x306B68267Deb7c5DfCDa3619E22E9Ca39C374f84","wormholeMessengerAddress":"0x118173028162C1b7c6Bf8488bd5dA2abd7c30F9D","childMockERC20Address":"0xE16adc7777B7C2a0d35033bd3504C028AB28EE8b"}
```

Comment:
Redeployed contracts to repeat the testing after addressing internal audit findings.